client/android/shadowsocks/src/main/java/org/amnezia/vpn/shadowsocks/core/Core.kt

```diff
@@ -161,7 +161,7 @@ object Core : Configuration.Provider {
                             NotificationManager.IMPORTANCE_LOW),
                     SubscriptionService.notificationChannel))
             notification.deleteNotificationChannel("service-nat")   // NAT mode is gone for good
-        }
+        } */
```

Comment:
Why you commented this part of code?

Reply:
Because we don't use notification channels in our app and I want to save maximum of original SS code.